README.md

```diff
@@ -154,6 +154,43 @@ vim.keymap.set("n", "glP", gitlab.publish_all_drafts)
 vim.keymap.set("n", "glD", gitlab.toggle_draft_mode)
 ```
 
+### Or if you are using [which-key](https://github.com/folke/which-key.nvim), here's what you can use:
```

Comment:
This should go in `doc/gitlab.nvim.txt` probably at the bottom of the keybindings section.